.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,8 +11,17 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **Setup**
-(Please provide relevant configs and/or SLS files (be sure to remove sensitive info).
+(Please provide relevant configs and/or SLS files (be sure to remove sensitive info. There is no general set-up of Salt.)
 
+Please be as specific as possible and give set-up details.
+
+
+- [ ] on-prem machine
+- [ ] VM (Virtualbox, KVM, etc. please specify)
```

Comment:
it might be VMs running on cloud services as well, we have to add an input field to add explicit comments to elaborate it.